ui/narrative/methods/run_metaSPAdes/spec.json

```diff
@@ -58,7 +58,8 @@
                 },
                 {
                     "input_parameter": "read_libraries",
-                    "target_property": "read_libraries"
+                    "target_property": "read_libraries",
+                    "target_type_transform": "list<ref>"
```

Comment:
What is the reasoning for this? Not clear to me why we'd cripple the app in the UI but allow anyone with knowledge of code cells to bypass the limitation.